units/commit_queue.go

```diff
@@ -336,98 +332,24 @@ func (j *commitQueueJob) processGitHubPRItem(ctx context.Context, cq *commitqueu
-				commitqueue.GithubContext,
-				pr.Base.User.GetLogin(),
-				pr.Base.Repo.GetName(),
-				pr.Head.GetRef(),
-				InvalidConfig,
-			)
-			update.Run(ctx)
-			j.AddError(update.Error())
-			j.logError(catcher.Resolve(), "invalid config file", nextItem)
-			j.AddError(sendCommitQueueGithubStatus(j.env, pr, message.GithubStateFailure, "can't make patch", ""))
-			j.dequeue(cq, nextItem)
-			return
-		}
-
-		if err = writePatchInfo(patchDoc, patchSummaries, patch); err != nil {
-			j.logError(err, "can't make patch", nextItem)
-			j.AddError(sendCommitQueueGithubStatus(j.env, pr, message.GithubStateFailure, "can't make patch", ""))
-			j.dequeue(cq, nextItem)
-			return
-		}
-
-		modulePRs, modulePatches, dequeue, err := getModules(ctx, githubToken, nextItem, projectConfig)
-		if err != nil {
-			j.logError(err, "can't get modules", nextItem)
-			if dequeue {
-				j.AddError(sendCommitQueueGithubStatus(j.env, pr, message.GithubStateFailure, "can't get modules", ""))
-				j.dequeue(cq, nextItem)
-			}
-			return
-		}
-		patchDoc.Patches = append(patchDoc.Patches, modulePatches...)
-
-		// populate tasks/variants matching the commitqueue alias
-		projectConfig.BuildProjectTVPairs(patchDoc, patchDoc.Alias)
-
-		if err = addMergeTaskAndVariant(patchDoc, projectConfig, projectRef, commitqueue.SourcePullRequest); err != nil {
-			j.logError(err, "can't set patch project config", nextItem)
-			event.LogCommitQueueEnqueueFailed(nextItem.Issue, err)
-			j.dequeue(cq, nextItem)
-			return
-		}
-
-		if err = patchDoc.Insert(); err != nil {
-			j.logError(err, "can't insert patch", nextItem)
-			j.dequeue(cq, nextItem)
-			j.AddError(sendCommitQueueGithubStatus(j.env, pr, message.GithubStateFailure, "can't make patch", ""))
-			return
-		}
-
-		// TODO EVG-14087 also do this when we finalize, with a different message
-		for _, modulePR := range modulePRs {
-			j.AddError(sendCommitQueueGithubStatus(j.env, modulePR, message.GithubStatePending, "added to queue", patchDoc.Id.Hex()))
-		}
-	} else {
-		var err error
-		patchDoc, err = patch.FindOneId(nextItem.PatchId)
-		if err != nil {
-			j.AddError(errors.Wrapf(err, "error finding patch '%s'", nextItem.Version))
-			return
-		}
-		if patchDoc == nil {
-			j.AddError(errors.Errorf("patch '%s' not found", nextItem.Version))
-			return
-		}
+	patchDoc, err := patch.FindOneId(nextItem.PatchId)
+	if err != nil {
+		j.AddError(errors.Wrapf(err, "error finding patch '%s'", nextItem.Version))
+		j.dequeue(cq, nextItem)
+		return
+	}
+	if patchDoc == nil {
+		j.AddError(errors.Errorf("patch '%s' not found", nextItem.Version))
+		j.dequeue(cq, nextItem)
+		return
+	}
+	projectConfig, _, err := model.GetPatchedProject(ctx, patchDoc, githubToken)
+	if err != nil {
+		j.logError(err, "problem getting patched project", nextItem)
+		j.dequeue(cq, nextItem)
+		j.AddError(sendCommitQueueGithubStatus(j.env, pr, message.GithubStateFailure, "can't get project config", ""))
```

Comment:
Shouldn't we send a status for the two cases above as well?

Reply:
yep done